src/lib/rpmostree-package.c

```diff
@@ -132,6 +132,28 @@ rpm_ostree_package_get_arch (RpmOstreePackage *p)
   return p->arch;
 }
 
+/**
+ * evr_cmp:
+ * @a: String representation of EVR
+ * @b: String representation of EVR
+ *
+ * Compares two string representations of EVR by calling rpmverCmp
+ *
+ * Returns: 1: a is newer than b
+ *          0: a and b are the same version
+ *         -1: b is newer than a
+ */
```

Comment:
Heh, this isn't a public API so we don't need nice docstrings like that, but obviously good. :)